static/src/javascripts/projects/common/modules/ui/subscription-banner.js

```diff
@@ -0,0 +1,163 @@
+// @flow
+
+import uniq from 'lodash/uniq';
+import { hasUserAcknowledgedBanner } from 'common/modules/ui/message';
+// import { trackNonClickInteraction } from 'common/modules/analytics/google';
```

Comment:
We might want to use this to record the component has been displayed.

```
ophan.record({
        component: componentName,
        action: interaction,
        value: interaction,
    });
    trackNonClickInteraction(interaction);
```

Reply:
Ah I see this is done otherwise for ophan, but maybe still worth putting in GA using just `trackNonClickInteraction(interaction)`

Reply:
add "subscription-banner : display" event to be used with trackNonClickInteraction